### DISCUSSION TITLE
few mqtt topics inbound connector(s?)

### DISCUSSION THREAD

**Discussion by pistoletov1974** [fluvio-community/fluvio#3932] [Q&A] [answered]
How to get data from multiple mqtt topics correctly - by running multiple connectors with their own configurations? Or am I describing several mqtt incoming topics in one configuration? I try first one 
```
cdk  deploy  start --config sql_tele.yaml --ipkg infinyon-sql-sink-0.3.3.ipkg
cdk  deploy  start --config tele_emqx.yaml --ipkg infinyon-mqtt-source-0.2.5.ipk
```
and receive error:
... checking package
Error: Failed to deploy from ipkg file

Caused by:
    **Text file busy (os error 26)**

**Comment by drc-infinyon** [comment]
@pistoletov1974 Are these for different sources of data?

@digikata - is this one example of what you referenced yesterday?

**Reply by pistoletov1974** [reply]
@drc-infinyon yes, different topics from the  same mqtt broker
I suppouse my situation described in [this issue](https://github.com/infinyon/fluvio/issues/3903)

**Reply by digikata** [reply]
That is correct, we have a PR that is being reviewed for merge that addresses this issue, should be coming up soon

**Reply by ajhunyady** [reply]
This issue has been resolved.

**Comment by sehz** [comment, marked as answer]
There is a limitation now with connector.  Can only start a single configuration in same directory.   Workaround is to create separate directory for each configuration

**Comment by ajhunyady** [comment]
@digikata, let's deploy the fix so @pistoletov1974 can confirm it works.

**Reply by digikata** [reply]
@pistoletov1974 this should be fixed with the merge of PR #3903.   If you would like to try it out, you can use the fvm to install our latest pre-release version:

`fvm install latest`

**Reply by ajhunyady** [reply]
@pistoletov1974 can you please check if the fix works?

**Reply by pistoletov1974** [reply]
@ajhunyady i need to update connectors only or full fluvio ? 
Can you give me few tips about this issue. [issue with connectror  build ](https://github.com/infinyon/sql-connector/issues/102)  I am not specialist in RUST. How can i build this connector?  I suppouse problem is with name of package.

**Reply by ajhunyady** [reply]
@pistoletov1974, the fix was in CDK. So, all you need to do is `fvm install latest` (fvm -> fluvio version manager). This command will download the `latest` binaries, such as fluvio and cdk and switch your environment to use them.

Once you are done testing, you can switch back to the `stable` by using `fvm switch stable`.

To check what versions you are running use `fvm list`.

**Reply by pistoletov1974** [reply]
@digikata @ajhunyady  I launched successfully few connectors from same directory-)

**Reply by ajhunyady** [reply]
Thank you for checking @pistoletov1974! The fix will be added to the next `stable` release targeted for next week.
